pootle/templates/user/profile.html

```diff
@@ -18,8 +18,8 @@ <h1 class="user-name">
         {{ object.display_name }}
       {% endif %}
     </h1>
-    {% if user.is_superuser or request.permissions.administrate %}
-    <p><a href="mailto:{{ object.email }}">{{ object.email }}</a></p>
+    {% if user.is_superuser or user.is_manager %}
```

Comment:
cool - i think my concern was basically that is_manager looks like is_superuser and is_staff but is less efficient at retrieving a result